int8 objnum

float32 type1
float32 confidence1
float32 distance1
float32 x1
float32 y1
float32 z1

float32 type2
float32 confidence2
float32 distance2
float32 x2
float32 y2
float32 z2

float32 type3
float32 confidence3
float32 distance3
float32 x3
float32 y3
float32 z3

float32 type4
float32 confidence4
float32 distance4
float32 x4
float32 y4
float32 z4

float32 type5
float32 confidence5
float32 distance5
float32 x5
float32 y5
float32 z5

float32 type6
float32 confidence6
float32 distance6
float32 x6
float32 y6
float32 z6

float32 type7
float32 confidence7
float32 distance7
float32 x7
float32 y7
float32 z7

float32 type8
float32 confidence8
float32 distance8
float32 x8
float32 y8
float32 z8

float32 type9
float32 confidence9
float32 distance9
float32 x9
float32 y9
float32 z9

float32 type10
float32 confidence10
float32 distance10
float32 x10
float32 y10
float32 z10

